include/board_pico_config.h

```diff
@@ -49,4 +49,7 @@
 
 #define PROBE_PRODUCT_STRING "Picoprobe (CMSIS-DAP)"
 
+// Host Reset Config
+#define PICOPROBE_RESET_CAPABLE 1
```

Comment:
I would prefer disabled by default. I would guess that for other users the feature would aid remote development of the probe firmware itself, but the vast majority will be doing uf2 drag-and-drop with a Pico or a Debug Probe and a standard release.

So, please move this definition to include/board_example_config.h with a suitable description.